tool/unwrap/unwrap.cwl

```diff
@@ -7,7 +7,7 @@ inputs:
     type: Any?
 expression: |
   ${
-    if (!inputs.input) {
+    if (!inputs.input === null) {
```

Comment:
There shouldn't be a `!` here, I don't think

Reply:
Fixed by #41. Thanks!